CONTRIBUTING.md

```diff
@@ -52,31 +52,26 @@ In addition to running npm install in the root of the project (to install the de
 # At project root
 npm install
 ```
+To test both samples you will need the following configured in your developer org:
 
-Then you need to setup the following environment variables
-
-```bash
-export CLIENT_ID={{yourAppClientId}}
-export OKTA_DOMAIN={{yourOktaOrgDomain}}
-```
-
-> **NOTE:** Use only the domain part of your org url while setting OKTA_DOMAIN environment.
+* A SPA application
```

Comment:
In our `README` we link out to "Setting up a SPA" - in this case lets link to our `README` for setup instructions.

Reply:
Good point. Updated.